server/services/notify/notifysubscriptions/diff2markdown_test.go

```diff
@@ -0,0 +1,26 @@
+package notifysubscriptions
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_reverse(t *testing.T) {
```

Comment:
Maybe some other tests would be helpful, things like the truncation function or the markdown generation from the diffs

Reply:
I created ticket https://community.mattermost.com/boards/workspace/qgsck6cts3fwpqwyjiupjm5cde/b4ctmm917q3rzdfi451qtuxjzcw/150d1464-6d59-4c39-810f-ee201439dbaf/c4p1m99neap8zdm59gb1u7361zw to add additional unit tests.  I'll get to this very soon, but I don't want to delay merge waiting for it.